src/storage/heap_file.c

```diff
@@ -21412,7 +21421,14 @@ heap_delete_home (THREAD_ENTRY * thread_p, HEAP_OPERATION_CONTEXT * context, boo
 	  rec_address.pgptr = context->home_page_watcher_p->pgptr;
 	  rec_address.vfid = &context->hfid.vfid;
 	  rec_address.offset = context->oid.slotid;
-	  heap_mvcc_log_delete (thread_p, &rec_address, RVHF_MVCC_DELETE_REC_HOME);
+	  if (prm_get_bool_value (PRM_ID_SUPPLEMENTAL_LOG) == true)
+	    {
+	      heap_mvcc_log_delete (thread_p, &rec_address, RVHF_MVCC_DELETE_REC_HOME, &context->home_recdes);
```

Comment:
The heap_mvcc_log_delete() function is also called in case of context->record_type == REC_ASSIGN_ADDRESS. In this case, how to deal with it?